recipes/ocgis/run_test.py

```diff
@@ -1,3 +1,13 @@
+import os
 from ocgis.test import run_simple
 
+# Set GDAL_DATA. This is done normally done by the activate script,
+# but this doesn't happen in the testing environment.
+if 'LIBRARY_PREFIX' in os.environ:  # Windows.
+    gdalData = os.path.join(os.environ['LIBRARY_PREFIX'], 'share', 'gdal')
+else:  # Linux/OS X.
+    gdalData = os.path.join(os.environ['PREFIX'], 'share', 'gdal')
```

Comment:
We are leaking GDAL implementation detail into this recipe, which isn't ideal. Maybe we can come up with a better solution in GDAL?

I've raised an issue in https://github.com/conda-forge/gdal-feedstock/issues/35.

Don't let it hold this PR up though - we can always go back and remove it!